Makefile.posix

```diff
@@ -7,14 +7,18 @@ LLVM_CONFIG ?= $(shell \
 	|| which /usr/local/opt/llvm@13/bin/llvm-config \
 	|| which llvm-config-11 \
 	|| which /usr/local/opt/llvm@11/bin/llvm-config \
+	|| which /usr/pkg/libexec/libLLVM/llvm-config \
 	)
 CFLAGS += $(shell $(LLVM_CONFIG) --cflags)
 CFLAGS += -gdwarf-4  # https://github.com/llvm/llvm-project/issues/56550
 LDFLAGS ?= $(shell $(LLVM_CONFIG) --ldflags --libs)
 
 ifeq ($(CC),cc)
 	# default c compiler --> use clang
-	CC := $(shell $(LLVM_CONFIG) --bindir)/clang
+	CC := $(shell \
+	which `$(LLVM_CONFIG) --bindir`/clang \
+	|| which clang \
```

Comment:
This is repeated in a couple places. Maybe make a new variable for it, or just use `$(CC)` everywhere.

Reply:
I’ll do that.